src/gucs.rs

```diff
@@ -0,0 +1,25 @@
+use pgrx::*;
+use std::ffi::CStr;
+
+pub static NEON_AUTH_JWK_RUNTIME_PARAM: &'static str = "neon.auth.jwk";
+pub static NEON_AUTH_JWK: GucSetting<Option<&'static CStr>> = GucSetting::<Option<&'static CStr>>::new(None);
+pub static NEON_AUTH_JWT_RUNTIME_PARAM: &'static str = "neon.auth.jwt";
+pub static NEON_AUTH_JWT: GucSetting<Option<&'static CStr>> = GucSetting::<Option<&'static CStr>>::new(None);
+
+pub fn init() {
+    GucRegistry::define_string_guc(
+        &NEON_AUTH_JWK_RUNTIME_PARAM,
+        "JSON Web Key (JWK) userd for JWT validation",
+        "Generated per connection by Neon local proxy",
+        &NEON_AUTH_JWK,
+        GucContext::Suset, // we should use GucContext::SuBackend but this breaks unit tests
```

Comment:
I still don't know:
- how to set it properly in unit tests
- how to set it properly while running `cargo pgrx run pg16`

so for now I'm using `Suset`

Reply:
I can write a manual test suite then that opens a new connection reach time. Should not be too much of a bother